Framework/DataObjects/src/PeaksWorkspace.cpp

```diff
@@ -838,73 +838,81 @@ void PeaksWorkspace::saveNexus(::NeXus::File *file) const {
   file->putAttr("units", "Not known"); // Units may need changing when known
   file->closeData();
 
-  // Bin Count column
-  file->writeData("column_7", binCount);
+  // Intensity over Sigma column
+  file->writeData("column_7", intensityOverSigma);
```

Comment:
This is in a `saveNexus` function - it there a corresponding `loadNexus` function? It is not in the same file and I am not sure if this is a breaking change for the save/load of peak WSs

Reply:
This is most definitely a breaking change and we need to ensure backwards compatibility with old files that dont save the `I/Sigma` column. It looks like the loading functionality is in [`LoadNexusProcessed.cpp`](https://github.com/mantidproject/mantid/blob/master/Framework/DataHandling/src/LoadNexusProcessed.cpp#L967).

Reply:
Okay, this is not that significant to this PR, so I will pull it out into a new one, to avoid delaying